homeassistant/components/airvisual/sensor.py

```diff
@@ -30,16 +38,26 @@
 MASS_PARTS_PER_BILLION = "ppb"
 VOLUME_MICROGRAMS_PER_CUBIC_METER = "µg/m3"
 
-GEOGRAPHY_SENSOR_KIND_LEVEL = "air_pollution_level"
-GEOGRAPHY_SENSOR_KIND_AQI = "air_quality_index"
-GEOGRAPHY_SENSOR_KIND_POLLUTANT = "main_pollutant"
+SENSOR_KIND_LEVEL = "air_pollution_level"
+SENSOR_KIND_AQI = "air_quality_index"
+SENSOR_KIND_POLLUTANT = "main_pollutant"
+SENSOR_KIND_BATTERY_LEVEL = "battery_level"
+SENSOR_KIND_HUMIDITY = "humidity"
+SENSOR_KIND_TEMPERATURE = "temperature"
+
 GEOGRAPHY_SENSORS = [
-    (GEOGRAPHY_SENSOR_KIND_LEVEL, "Air Pollution Level", "mdi:gauge", None),
-    (GEOGRAPHY_SENSOR_KIND_AQI, "Air Quality Index", "mdi:chart-line", "AQI"),
-    (GEOGRAPHY_SENSOR_KIND_POLLUTANT, "Main Pollutant", "mdi:chemical-weapon", None),
+    (SENSOR_KIND_LEVEL, "Air Pollution Level", "mdi:gauge", None),
+    (SENSOR_KIND_AQI, "Air Quality Index", "mdi:chart-line", "AQI"),
+    (SENSOR_KIND_POLLUTANT, "Main Pollutant", "mdi:chemical-weapon", None),
 ]
 GEOGRAPHY_SENSOR_LOCALES = {"cn": "Chinese", "us": "U.S."}
 
+NODE_PRO_SENSORS = [
+    (SENSOR_KIND_BATTERY_LEVEL, "Battery", "battery", UNIT_PERCENTAGE),
```

Comment:
Please use the device class constants imported from const.py, eg `DEVICE_CLASS_BATTERY`.